docs/design/ui-kit.md

```diff
@@ -1,33 +1,46 @@
 ---
-title: Docker extension design
+title: Docker UI Kit
 description: Docker extension design
 keywords: Docker, extensions, design
 ---
 
-## Docker UI Kit
+# Docker UI Kit
 
-Our Design System is currently being developed. In the meantime, you will find some basic styles (color, typography) and components in this [Docker Extensions Styleguide](https://www.figma.com/file/U7pLWfEf6IQKUHLhdateBI/Docker-Design-Guidelines?node-id=1%3A28771). Using the components and styles as per the Docker Extensions Styleguide will ensure that your extension meets the [level AA accessibility standard.](https://www.w3.org/WAI/WCAG2AA-Conformance)
+Our Design System is a constantly evolving set of specifications that aim to ensure visual consistency across Docker products, and meet [level AA accessibility standards](https://www.w3.org/WAI/WCAG2AA-Conformance). We've opened parts of it to extension authors, documenting basic styles (color, typography) and components. See: [Docker Extensions Styleguide](https://www.figma.com/file/U7pLWfEf6IQKUHLhdateBI/Docker-Design-Guidelines?node-id=1%3A28771).
 
-<br>
+We require extensions to match the wider Docker Desktop UI to a certain degree, and reserve the right to make this stricter in the future.
```

Comment:
Bit of a hard take, but I think it's better for them to be told here than at quality-control review. @amyb12345, @mel-manohar, does this wording seem okay?

Reply:
Completely agree Mark. I have added this in design-guidelines.md as well. "we strongly recommend that you follow these guidelines when building extensions. We will be reviewing and approving your Marketplace publication based on these requirements."